instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/internal/instrumentation/scheduling/SpringSchedulingInstrumentationAspect.java

```diff
@@ -39,7 +39,7 @@
  */
 @Aspect
 final class SpringSchedulingInstrumentationAspect {
-  public static final String INSTRUMENTATION_NAME = "io.opentelemetry.spring-scheduling-3.1";
+  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.spring-boot-autoconfigure";
```

Comment:
Instrumentation name should align with the module that contains the instrumentation code. It would probably be better to move this code elsewhere. cc @zeitlinger